app/assets/stylesheets/components/cards.scss.erb

```diff
@@ -33,3 +33,17 @@
 	&.mastercard { background-position: 64px; }
 	&.discovercard { background-position: 96px; }
 }
+
+<%# currently only the /components/card-form.es6 form for the widget uses this  %>
```

Comment:
It would be pretty simple to include this change for the recurring-donations version of this as well. It's a tradeoff of making changes to only one part of the app at a time vs. keeping the two sets of code for the same functionality in sync.

Reply:
As long as it doesn't break anything, let's do that in a separate PR.